h2/src/test/org/h2/test/unit/TestSubqueryPerformanceOnLazyExecutionMode.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2004-2018 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (http://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.test.unit;
+
+import org.h2.command.dml.SetTypes;
+import org.h2.test.TestBase;
+import org.junit.Assert;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Test subquery performance with lazy query execution mode {@link SetTypes#LAZY_QUERY_EXECUTION}.
+ */
+public class TestSubqueryPerformanceOnLazyExecutionMode extends TestBase {
```

Comment:
Is there any reason to extend `TestBase` instead of `TestDb` here?

(Also you can override `isEnabled()` if your test case should be restricted to some testing modes, like only in-memory or something like it.)

Reply:
Fixed.